src/main/java/de/tum/in/www1/artemis/service/metis/conversation/ChannelService.java

```diff
@@ -137,6 +141,15 @@ public Channel createChannel(Course course, Channel channel, Optional<User> crea
         return savedChannel;
     }
 
+    public Set<User> registerUsersToChannel(Boolean addAllStudents, Boolean addAllTutors, Boolean addAllInstructors, List<String> usersLoginsToRegister, Course course,
```

Comment:
is there are reason, the parameters are Booleans and not booleans, can they ever be null?

Reply:
no, will change it to `boolean` 👍 .